inc/admin/metabox/main.php

```diff
@@ -58,11 +45,7 @@ private function add_layout_controls() {
 			)
 		);
 
-		$position_default = ( self::is_new_page() || self::is_checkout() ) ? 'full-width' : 'default';
-
-		if ( $this->new_skin ) {
-			$position_default = 'default';
-		}
+		$position_default = 'default';
```

Comment:
I'm not sure why this was changed. The condition should remain the same.

Reply:
This changed because below I removed the condition that checked for the `new_skin` and from the code if it was `new_skin` it would always have the value of `default`.